test/acceptance_with_go_client/named_vectors_tests/test_suits/named_vectors_objects_mixed.go

```diff
@@ -209,6 +212,54 @@ func testCreateSchemaWithMixedVectorizers(host string) func(t *testing.T) {
 					assert.Equal(t, vector, models.Vector(afterUpdateVectors[targetVector]))
 				}
 			})
+
+			t.Run("check BYOV vector names for existence when inserting", func(t *testing.T) {
+				_, err = client.Data().Creator().
+					WithClassName(className).
+					WithID(id3).
+					WithProperties(map[string]interface{}{
+						"text": "Lorem ipsum dolor sit amet",
+					}).
+					WithVectors(models.Vectors{
+						"non_existent_vector": []float32{0.1, 0.2, 0.3},
+					}).
+					Do(ctx)
+				require.Error(t, err)
+				var clientError *fault.WeaviateClientError
+				require.True(t, errors.As(err, &clientError))
+				require.Equal(t, 422, clientError.StatusCode)
+				require.Contains(t, clientError.Msg, fmt.Sprintf("collection %s does not have configuration for vector non_existent_vector", className))
```

Comment:
perfect timing with the PR -- we've just ran into this same issue with @g-despot and agreed that the error message could've been better